docs/README.md

```diff
@@ -57,3 +59,4 @@ The subdomain used for the project can be configured via the `modules.local-serv
 * `composer server db` - Logs into MySQL on the DB container.
   * `composer server db info` - Print MySQL connection details.
   * `composer server db sequel` - Opens a connection to the database in [Sequel Pro](https://sequelpro.com).
+* `composer server sync` - Sync files from `content/uploads` to the s3 container.
```

Comment:
I think we need a more clearer name than `sync`, needs to speak to uploads. `import-uploads` maybe? It doesn't appear to be a sync operation, because it's only one way, with overriting enabled?

Reply:
It’s essentially `rsync`, so not sure sync necessarily implies 2 way either but agree on the naming. Import implies the specific direction this is running in so I think that makes sense.

Reply:
Updated the command name